vizro-core/src/vizro/cards/_cards.py

```diff
@@ -0,0 +1,119 @@
+"""Module containing default card components."""
+
+from typing import Callable, Optional
+
+import dash_bootstrap_components as dbc
+import pandas as pd
+from dash import dcc, get_relative_path, html
+
+from vizro.models.types import capture
+
+
+# LQ: The data frame is not used in the function, but needs to be provided here. I guess that is unavoidable for now?
```

Comment:
@antonymilne - I've left several **LQ** throughout the files which are open questions I currently have

Reply:
I am wondering if we should actually have a complete new model `DynamicCard` with a figure attribute only. Feels cleaner to me. Let's discuss tomorrow 👍